mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -2455,15 +2493,15 @@ msgstr ""
 "Tienda de Arte!\n"
 "Desde que nos cortaron el presupuesto hemos tenido que vender las obras de "
 "arte en lugar de solo mostrarlas.\n"
-"La entrada cuesta 50${{currency}}. Va a los dividendos anuales de la compañía."
+"La entrada cuesta 50$. Va a los dividendos anuales de la compañía."
```

Comment:
You should use the old translation. `${{currency}}` will be replaced with the proper money symbol of the campaign, with may be different from the dollar sign.

Reply:
```suggestion
"La entrada cuesta 50${{currency}}. Va a los dividendos anuales de la compañía."
```